Makefile

```diff
@@ -135,7 +141,8 @@ format: .bin/goimports docs/node_modules node_modules
 # Build local docker image
 .PHONY: docker
 docker:
-		DOCKER_BUILDKIT=1 docker build -f .docker/Dockerfile-build --build-arg=COMMIT=$(VCS_REF) --build-arg=BUILD_DATE=$(BUILD_DATE) -t oryd/kratos:latest .
+		@echo "Building Image 'oryd/kratos:$(IMAGE_TAG)'"
+		DOCKER_BUILDKIT=1 docker build -f .docker/Dockerfile-build --build-arg=COMMIT=$(VCS_REF) --build-arg=BUILD_DATE=$(BUILD_DATE) -t oryd/kratos:$(IMAGE_TAG) .
```

Comment:
@tricky42 this will use the wrong docker image. We are publishing the Dockerfile-alpine image. The Dockerfile-build image is primarily there to make it easy for local development :)

You should be able to build alpine like this:

touch kratos
docker build -f .docker/Dockerfile-alpine
rm kratos

Reply:
I looked at it and the only difference between the two resulting issues is the missing 
```
RUN apk add -U --no-cache ca-certificates
```
for the resulting image. But you are right, in order to prevent that over time we change the published images we should just use the `Dockerfile-alpine` image. 

I will adjust the way we build the container to use `Dockerfile-alpine`.